### ISSUE TITLE
bug: Can't signin after reset password

### ISSUE THREAD

**Issue by shensven** [logto-io/logto#2060]
### Describe the bug

Access Logto's console with admin account, reset user's password, can't sign in again with original password or new one.

### Context

OS: `macOS 12.6`

Environment: 
```yaml
version: '3.9'
services:
  logto:
    image: ghcr.io/logto-io/logto:1.0.0-beta.10
    ports:
      - 3001:3001
    environment:
      - ALL_YES=1
      - NO_INQUIRY=0
      - TRUST_PROXY_HEADER=1
      - USER_DEFAULT_ROLE_NAMES=user,good_user
      - DB_URL_DEFAULT=${HASURA_GRAPHQL_DATABASE_URL}
      - ENDPOINT
```

Logto version: `1.0.0-beta.10`

Node version: `v16.17.0`

Postgres version: `13.7`

Command of starting Logto: `docker-compose up`

### Screenshots

<img width="1212" alt="1" src="https://user-images.githubusercontent.com/16205980/194699697-7d68ca5f-4467-4fb7-8cd9-836b9b042e06.png">
<img width="879" alt="2" src="https://user-images.githubusercontent.com/16205980/194699699-17203851-434c-4075-9687-0d9f286dff43.png">

**Comment by wangsijie** [comment]
Thanks for the feedback, I am taking a look on that, this is a bug and will be fixed soon.

**Comment by wangsijie** [comment]
It is "base64 encoded" by mistake, and is now fixed in #2063 

But the docker image will not be updated until our next release, for now, you can either use the `master` code to build your own image, or manually "base64 decode" the password shown in the dialog.

Let me know if you have further questions.
